src/index.html

```diff
@@ -1,12 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content" >
+    <div class="calendar page__calendar calendar--start-day--sunday calendar--month-length--28">
```

Comment:
```suggestion
    <div class="calendar page__calendar calendar--sunday calendar--28">
```
Можно записывать короче